packages/rrweb/src/record/index.ts

```diff
@@ -345,15 +335,13 @@ function record<T = eventWithTime>(
       mirror,
       win: window,
       mutationCb: (p: canvasMutationParam) =>
-        wrappedEmit(
-          wrapEvent({
-            type: EventType.IncrementalSnapshot,
-            data: {
-              source: IncrementalSource.CanvasMutation,
-              ...p,
-            },
-          }),
-        ),
+        wrappedEmit({
+          type: EventType.IncrementalSnapshot,
+          data: {
+            source: IncrementalSource.CanvasMutation,
+            ...p,
```

Comment:
more a sidenote and unrelated to this PR, but I wonder if we could/should also make a change to all of these to avoid spreading these, and instead just set the source on the data 🤔 may help a little bit with memory usage, but not sure if worth it...

Reply:
We could run benchmark suite and see if there's anything noticable.